package.json

```diff
@@ -1,19 +1,16 @@
 {
-  "name": "hydrolysis",
-  "version": "1.24.1",
-  "description": "Breaks polymers into monomers",
-  "homepage": "https://github.com/Polymer/hydrolysis",
-  "bugs": "https://github.com/Polymer/hydrolysis/issues",
+  "name": "polymer-analyzer",
+  "version": "2.0.0-alpha.1",
+  "description": "Static analysis for Web Components",
+  "homepage": "https://github.com/Polymer/polymer-analyzer",
+  "bugs": "https://github.com/Polymer/polymer-analyzer/issues",
   "license": "BSD-3-Clause",
   "repository": {
     "type": "git",
-    "url": "https://github.com/Polymer/hydrolysis.git"
+    "url": "https://github.com/Polymer/polymer-analyzer.git"
   },
   "main": "index.js",
   "files": [
-    "hydrolysis-analyzer.html",
-    "hydrolysis.html",
-    "hydrolysis.js",
     "index.js",
```

Comment:
Should add custom_typings/ to this array though, so downstream installers get all the definitions they need.